src/main/java/org/spongepowered/api/event/entity/ai/AITaskEvent.java

```diff
@@ -77,6 +77,14 @@
     }
 
     /**
+     * Fired when an {@link AITask} is updated every
+     */
+    @FactoryCodeCheck(errorMessage = "The target entity '%s' is not the owner of the goal '%s'!")
+    interface Tick extends AITaskEvent {
```

Comment:
Not too confident on this event....because of the incredible amount of spam. Put this into another PR please.